src/App.vue

```diff
@@ -225,6 +244,30 @@ ul ul {
   }
 }
 
+.block-number {
+  &-link {
+    display: flex;
+    align-items: center;
+    color: var(--s-color-status-success);
+    font-size: var(--s-font-size-small);
+    text-decoration: none;
+    font-weight: 300;
+    position: absolute;
```

Comment:
![block-num](https://user-images.githubusercontent.com/53777036/139636316-583e87ca-61e2-4ace-a4ec-38217c6c6a34.jpg)
maybe show it on screen resolutions from `tablet`?

Reply:
fixed. on tablets it doesn't show